src/main/java/net/sf/jabref/exporter/OOCalcDatabase.java

```diff
@@ -67,7 +66,7 @@ public OOCalcDatabase(BibtexDatabase bibtex, Set<String> keySet) {
             }
         }
 
-        entries = new SortedList(entryList, new FieldComparatorStack(comparators));
+        entries = new SortedList<BibtexEntry>(entryList, new FieldComparatorStack<BibtexEntry>(comparators));
```

Comment:
You just need `<>` in the right side of a statement. This is called the diamond operator.